src/hypertable.c

```diff
@@ -1465,3 +1465,63 @@ ts_hypertable_create(PG_FUNCTION_ARGS)
 	cache_release(hcache);
 	PG_RETURN_VOID();
 }
+
+/* Used as a tuple found function */
+static bool
+rename_hypertable_schema_name(TupleInfo *ti, void *data)
+{
+	char	  **schema_names = (char **) data;
```

Comment:
Should probably `const` this strings to indicate that they shouldn't be modified.